lib/hooks/useAuthorizationPolicyById/useAuthorizationPolicyById.js

```diff
@@ -0,0 +1,28 @@
+import { useQuery } from 'react-query';
+
+import { useOkapiKy, useNamespace } from '@folio/stripes/core';
+
+import { POLICIES_API } from '../../constants';
+
+const DEFAULT_DATA = {};
+
+export const useAuthorizationPolicyById = (id, options = {}) => {
+  const { enabled = true, tenantId, ...otherOptions } = options;
+
+  const ky = useOkapiKy({ tenant: tenantId });
+
+  const [namespace] = useNamespace({ key: 'ui-authorization-policy' });
```

Comment:
Looks like key:'ui-authorization-policy' is redundant here. 'ui-authorization-policy'  will be settled by default. 
```suggestion
  const [namespace] = useNamespace();
```